edge/pkg/eventbus/event_bus.go

```diff
@@ -34,14 +34,13 @@ type eventbus struct {
 }
 
 func init() {
-	mode := config.CONFIG.GetConfigurationByKey("mqtt.mode")
-	if mode == nil {
-		mode = internalMqttMode
-	}
-	if mode.(int) > externalMqttMode || mode.(int) < internalMqttMode {
+	mode, err := config.CONFIG.GetValue("mqtt.mode").ToInt()
+	if err != nil {
+		panic(err)
```

Comment:
Is there a need to panic here ? If mode is not specified in config file, we can safely start with internalMqttMode

Reply:
OK